code/datums/antagonists/antag_datum.dm

```diff
@@ -73,6 +102,12 @@
 	var/datum/antagonist/D = new datum_type()
 	. = D.give_to_body(src)
 
+/mob/living/proc/lose_antag_datum(datum_type) //tries to remove an antagonist datum from a mob
```

Comment:
should have the same 'check_subtypes' arg as below, so that if something uses subtypes for specific effects you don't need to try and lose each type, just the main type.